src/execute/engine.ts

```diff
@@ -276,3 +277,30 @@ export function projectIgnoreGlobs(dir: string) {
     gitignoreEntries(dir).map((ignore) => `**/${ignore}**`),
   );
 }
+
+export const engineCommand = new Command()
+  .name("engine")
+  .description(
+    `Access functionality specific to quarto's different rendering engines.`,
+  )
+  .action(() => {
+    engineCommand.showHelp();
+    Deno.exit(1);
+  });
+
+kEngines.forEach((engine, name) => {
+  if (engine.populateCommand) {
+    const engineSubcommand = new Command();
+    // fill in some default behavior for each engine command
+    engineSubcommand
+      .description(
+        `Access functionality specific to the ${name} rendering engine.`,
+      )
```

Comment:
I like this. I think that we should let engines have a method to build a response, but that can happen in a different PR.